ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/representative-journey.spec.ts

```diff
@@ -121,19 +123,19 @@ describe('RepresentativeJourney', () => {
   it('should raise an error on unexpected failure transition', () => {
     givenUserIsAtStep(Steps.AboutVideoHearings);
     expect(() => whenFailingTheStep())
-      .toThrowError(`Missing/unexpected failure for step: ${RepresentativeJourneySteps[Steps.AboutVideoHearings]}`);
+      .toThrowError(`Missing/unexpected failure for step: ${Steps.AboutVideoHearings}`);
   });
 
   it('should raise an error on missing transition', () => {
     givenUserIsAtStep(Steps.ThankYou);
     expect(() => whenProceeding())
-      .toThrowError(`Missing transition for step: ${RepresentativeJourneySteps[Steps.ThankYou]}`);
+      .toThrowError(`Missing transition for step: ${Steps.ThankYou}`);
   });
 
   it('should goto video app if there are no upcoming hearings', () => {
     journey.forSuitabilityAnswers(suitabilityAnswers.noUpcomingHearings);
     journey.jumpTo(Steps.AboutVideoHearings);
-    expectStep(redirected).toBe(Steps[Steps.GotoVideoApp]);
+    expectStep(redirected).toBe(step(Steps.GotoVideoApp));
```

Comment:
I wonder if it's worthwhile to replace this now, can you test what happens if you simply do:
```js
expect(redirected).toBe(Steps.GotoVideoApp);
```

Because that `expectStep` and `step` basically just converts it to string but we might be able to check on reference

Reply:
Made changes